### PR TITLE
Load GetStats script to Jitsi/Mediasoup through test

### PR DESCRIPTION
**Pull request by karen-saldanha**

### CODE REVIEW

File: KITE-Jitsi-Test/src/main/java/io/cosmosoftware/kite/jitsi/steps/LoadGetStatsStep.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import javax.json.JsonValue;
+
+import static io.cosmosoftware.kite.util.TestUtils.executeJsScript;
+import static io.cosmosoftware.kite.util.TestUtils.waitAround;
+
+public class LoadGetStatsStep extends TestStep {
+    private final String pathToGetStats;
+    private final String testName;
+    private final String testId;
+    private final JsonValue logstashUrl;
```
Comment (ManuCosmo) [inline KITE-Jitsi-Test/src/main/java/io/cosmosoftware/kite/jitsi/steps/LoadGetStatsStep.java]: I had changed them to String on Jitsi.
Pls check and follow my changes in https://github.com/CoSMoSoftware/KITE-Sample-Tests/commit/7e717d9e064d2dfb7f10a73a2ddc74d81b39fdca

File: KITE-Jitsi-Test/src/main/java/io/cosmosoftware/kite/jitsi/steps/LoadGetStatsStep.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    private final String sfu;
+    private final JsonValue statsPublishingInterval;
+    private final String username;
+    private final String room;
```
Comment (ManuCosmo) [inline KITE-Jitsi-Test/src/main/java/io/cosmosoftware/kite/jitsi/steps/LoadGetStatsStep.java]: room is a constant, so it should be declared as such.
`
private final String ROOM_JS_OBJ = "APP.conference._room.options.name";`

File: KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/steps/LoadGetStatsStep.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+public class LoadGetStatsStep extends TestStep {
+    private final String pathToGetStats;
+    private final String testName;
+    private final String testId;
+    private final JsonValue logstashUrl;
+    private final String sfu;
+    private final JsonValue statsPublishingInterval;
+    private final String username;
+    private final String room;
```
Comment (ManuCosmo) [inline KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/steps/LoadGetStatsStep.java]: room is a constant, so it should be declared as such.
`
private final String ROOM_JS_OBJ =  "window.location.search.split(\"?roomId=\")[1]";
`

File: KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/steps/LoadGetStatsStep.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        getStatsScript = publishingSplit[0] + "testStats.startPublishing(" + statsPublishingInterval + ")" + publishingSplit[1];
+
+        String[] pcSplit = getStatsScript.split("window\\.pc");
+        getStatsScript = pcSplit[0] + "window.PC1" + pcSplit[1] + "window.PC1" + pcSplit[2];
+
+        String[] remotePcSplit = getStatsScript.split("window\\.remotePc\\[0]");
+        getStatsScript = remotePcSplit[0] + "window.PC2" + remotePcSplit[1];
+
+        String[] remotePc2Split = getStatsScript.split("window\\.remotePc");
+        getStatsScript = remotePc2Split[0] + "window.PC2" + remotePc2Split[1];
```
Comment (ManuCosmo) [inline KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/steps/LoadGetStatsStep.java]: only 2 remotePC?

Comment (karen-saldanha) [reply KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/steps/LoadGetStatsStep.java]: mediasoup only has 2 peer connection variables -> window.PC1 (sent data) and window.PC2 (received data)

**Comment by ManuCosmo** [comment]
pls resolve the conflicts

File: KITE-Jitsi-Test/src/main/java/io/cosmosoftware/kite/jitsi/KiteJitsiTest.java [hunk 5]
```diff
@@ -5,13 +5,45 @@
...
 import static org.webrtc.kite.Utils.getStackTrace;
 
+
+import javax.json.JsonObject;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Random;
+
 public class KiteJitsiTest extends KiteBaseTest {
+  public static String url = "https://meet.jit.si";
```
Comment (ManuCosmo) [inline KITE-Jitsi-Test/src/main/java/io/cosmosoftware/kite/jitsi/KiteJitsiTest.java]: it this used?

File: KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/KiteMediasoupTest.java [hunk 6]
```diff
@@ -12,17 +13,35 @@
...
+  private String pathToGetStatsSdk;
 
   @Override
   protected void payloadHandling() {
     super.payloadHandling();
     JsonObject jsonPayload = this.payload;
     if (jsonPayload != null) {
+      getStatsSdk = this.payload.getJsonObject("getStatsSdk");
+      testName = this.name;
+      testId = getStatsSdk.getString("testId");
```
Comment (ManuCosmo) [inline KITE-Mediasoup-Test/src/main/java/io/cosmosoftware/kite/mediasoup/KiteMediasoupTest.java]: that will throw a nullPointerException if there is no "getStatsSdk" in the config

**Pull request merged**